tailscale/resource_logstream_configuration.go

```diff
@@ -101,6 +176,11 @@ func resourceLogstreamConfigurationRead(ctx context.Context, d *schema.ResourceD
 	client := m.(*tsclient.Client)
 
 	logstream, err := client.Logging().LogstreamConfiguration(ctx, tsclient.LogType(d.Id()))
+	if err != nil && tsclient.IsNotFound(err) {
+		d.SetId("")
+		return nil
+	}
```

Comment:
Oh right! This is an unrelated bugfix I snuck in. Or at least I think it's a bugfix.

If you delete your Terraform-created logstream configuration out-of-band (e.g. in the admin console), without this fix, we fail with a 404 error on `plan`. I think probably we should instead detect that the config has been deleted and report that, right?

Reply:
I'll defer to @mpminardi on this, as I don't know what the usual convention is with Terraform.

Reply:
Whoops good catch on this! Yes I think if we get a 404 here we do want to report this / remove the resource from state. Have filed [this](https://github.com/tailscale/corp/issues/26072) ticket to track doing this across the board as I imagine we are missing this in a number of places.